docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
The Makefile and make.bat only existed in the Py manual. Not sure if we need them.